helper/strutil/strutil.go

```diff
@@ -174,6 +174,16 @@ func ParseArbitraryStringSlice(input string, sep string) []string {
 	return ret
 }
 
+// TrimStrings takes a slice of strings and returns a slice of strings
+// with trimmed spaces
+func TrimStrings(items []string) []string {
+	var ret []string
```

Comment:
Note that you can make not with length but with capacity. Then you can still append without having to keep track of indexes but avoid the resize.